.github/workflows/front-prod-deploy.yml

```
@@ -40,7 +40,17 @@ jobs:
           host: ${{ secrets.PROD_HOST }}
           key: ${{ secrets.PROD_RSA_PRIVATE_KEY }}
           script:
-            tar -xvf /home/ubuntu/jujeol-jujeol/frontend/frontend/deploy.tar &&
-            rm -rf /home/ubuntu/jujeol-jujeol/frontend/deploy/* &&
+            rm -rf /home/ubuntu/jujeol-jujeol/frontend/frontend/build &&
+            tar -xvf /home/ubuntu/jujeol-jujeol/frontend/frontend/deploy.tar -C /home/ubuntu/jujeol-jujeol/frontend/frontend
+            
+      - name: Change build files
+        uses: appleboy/ssh-action@master
+        with:
+          username: ubuntu
+          host: ${{ secrets.PROD_HOST }}
+          key: ${{ secrets.PROD_RSA_PRIVATE_KEY }}
+          script:
+            rm -rf /home/ubuntu/jujeol-jujeol/frontend/deploy &&
```

안녕하세요 소롱~ 이 부분 다음과 같이 수정이 필요할 것 같아요.

```suggestion
            rm -rf /home/ubuntu/jujeol-jujeol/frontend/deploy/* &&
```

현재 nginx를 도커로 실행시키고 있는데, 폴더자체를 삭제해버리면 폴더의 주소값이 달라서 마운트가 해제되어

운영환경에서 프론트엔드 빌드 코드를 찾지 못하는 문제가 발생하네요!

---

아하 몰랐네요 수정할게요!